package.json

```diff
@@ -13,6 +13,7 @@
     "lint"
   ],
   "dependencies": {
+    "@react-native-community/async-storage": "^1.6.3",
```

Comment:
Why to we need `async-storage` from 3rd party?

Reply:
Because AsyncStorage in 'react-native' will be removed in the near future and they are warning it